### PR TITLE
Release Notes for 8.3.1

### PR DESCRIPTION
**Pull request by robbavey**
No user facing changes in this release

### CODE REVIEW

**Review by kilfoyle — APPROVED** [review]
LGTM! 🚀

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -20,9 +21,28 @@ This section summarizes the changes in the following releases:
...
 [[logstash-8-3-0]]
 === Logstash 8.3.0 Release Notes
 
+[[known-issue-8-3-0]]
+==== Known issue
+
+An event serialization bug was discovered, which causes an issue when trying to read dead letter queues created
+in previous versions of Logstash.
+
+We recommend not upgrading to Logstash 8.3.0 if you are using the dead letter queue.
```
Comment (kaisecheng) [inline docs/static/releasenotes.asciidoc:43]: should we also mention persistent queue?

Comment (jsvd) [inline docs/static/releasenotes.asciidoc:43]:
```suggestion
We recommend not upgrading to Logstash 8.3.0 if you are using Dead Letter or Persistent Queues.
```

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -20,9 +21,28 @@ This section summarizes the changes in the following releases:
...
+  If you're using dead letter queue we recommend that you do not use Logstash 8.3.0 and upgrade to 8.3.1.
+  
+
 [[logstash-8-3-0]]
 === Logstash 8.3.0 Release Notes
 
+[[known-issue-8-3-0]]
+==== Known issue
+
+An event serialization bug was discovered, which causes an issue when trying to read dead letter queues created
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:40]:
```suggestion
An event serialization bug was discovered, which causes an issue when trying to read Dead Letter or Persistent Queues created
```

**Review by jsvd — CHANGES_REQUESTED** [review]

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -20,9 +21,28 @@ This section summarizes the changes in the following releases:
...
+  If you're using dead letter queue we recommend that you do not use Logstash 8.3.0 and upgrade to 8.3.1.
+  
+
 [[logstash-8-3-0]]
 === Logstash 8.3.0 Release Notes
 
+[[known-issue-8-3-0]]
+==== Known issue
+
+An event serialization bug was discovered, which causes an issue when trying to read Dead Letter or Persistent Queues created
```
Comment (kilfoyle) [inline docs/static/releasenotes.asciidoc:40]:
```suggestion
An event serialization bug was discovered, which causes an issue when trying to read dead letter or persistent queues created
```

File: docs/static/releasenotes.asciidoc [hunk 4]
```diff
@@ -20,9 +21,28 @@ This section summarizes the changes in the following releases:
...
 [[logstash-8-3-0]]
 === Logstash 8.3.0 Release Notes
 
+[[known-issue-8-3-0]]
+==== Known issue
+
+An event serialization bug was discovered, which causes an issue when trying to read dead letter or persistent queues created
+in previous versions of Logstash.
+
+We recommend not upgrading to Logstash 8.3.0 if you are using Dead Letter or Persistent Queues.
```
Comment (kilfoyle) [inline docs/static/releasenotes.asciidoc:43]:
```suggestion
We recommend not upgrading to Logstash 8.3.0 if you are using dead letter or persistent queues.
```

File: docs/static/releasenotes.asciidoc [hunk 5]
```diff
@@ -20,9 +21,28 @@ This section summarizes the changes in the following releases:
...
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-3-1]]
+=== Logstash 8.3.1 Release Notes
+
+[[notable-8.3.1]]
+==== Notable issues fixed
+
+* We fixed an event serializing incompatibility introduced in 8.3.0 https://github.com/elastic/logstash/pull/14314[#14314]
+  If you're using dead letter queue we recommend that you do not use Logstash 8.3.0 and upgrade to 8.3.1.
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:31]:
```suggestion
  If you're using dead letter queues or persistent queues we recommend that you do not use Logstash 8.3.0 and upgrade to 8.3.1.
```

**Review by jsvd — APPROVED** [review]
LGTM

**Comment by andsel** [comment]
I think this should be forward ported to `main` branch

**Pull request merged**